README.md

```diff
@@ -0,0 +1,135 @@
+# Microsoft Dynamics 365 CRM dbt Package ([Docs](https://fivetran.github.io/dbt_dynamics_365_crm/))
+
+<p align="left">
+    <a alt="License"
+        href="https://github.com/fivetran/dbt_fivetran_log/blob/main/LICENSE">
+        <img src="https://img.shields.io/badge/License-Apache%202.0-blue.svg" /></a>
+    <a alt="dbt-core">
+        <img src="https://img.shields.io/badge/dbt_Core™_version->=1.3.0_<2.0.0-orange.svg" /></a>
+    <a alt="Maintained?">
+        <img src="https://img.shields.io/badge/Maintained%3F-yes-green.svg" /></a>
+    <a alt="PRs">
+        <img src="https://img.shields.io/badge/Contributions-welcome-blueviolet" /></a>
+    <a alt="Fivetran Quickstart Compatible"
+        href="https://fivetran.com/docs/transformations/dbt/quickstart">
+        <img src="https://img.shields.io/badge/Fivetran_Quickstart_Compatible%3F-yes-green.svg" /></a>
+</p>
+
+## What does this dbt package do?
+
+This package models Microsoft Dynamics 365 CRM data from [Fivetran's connector](https://fivetran.com/docs/applications/microsoft-dynamics/dynamics365crm). It uses data in the format described by [this ERD](https://fivetran.com/docs/applications/microsoft-dynamics/dynamics365crm#schemainformation).
+
+The main focus of the package is to enhance the Microsoft Dynamics 365 CRM data by adding human-readable labels for fields that store coded values (e.g., integer codes or option sets). This package integrates stringmaps into the source tables, translating codes into meaningful labels.
+
+<!--section="dynamics_365_crm_transformation_model"-->
+The following table provides a detailed list of all models materialized within this package by default.
+> TIP: See more details about these tables in the package's [dbt docs site](https://fivetran.github.io/dbt_dynamics_365_crm/#!/overview).
+
+| **Table** | **Description** |
+| --------- | --------------- |
+| [`account`](https://fivetran.github.io/dbt_dynamics_365_crm/#!/model/model.dynamics_365_crm.account) | Model representing accounts in Dynamics 365 CRM, enriched with human-readable column names for fields with corresponding stringmap values. |
```

Comment:
```suggestion
| [`account`](https://fivetran.github.io/dbt_dynamics_365_crm/#!/model/model.dynamics_365_crm.account) | Model representing accounts in Dynamics 365 CRM, enriched with human-readable column names for fields with corresponding stringmap created as `<field_name>_label`. |
```

Reply:
I'm open to a different way to say this, but we should make sure customers are aware of what to look for regarding the stringmap fields. Please make this same update to the rest of the descriptions in this README and the docs as well.

Reply:
This makes sense to me! Updated.